### DISCUSSION TITLE
How do i update OpenCore?

### DISCUSSION THREAD

**Discussion by kingsvell** [Coopydood/ultimate-macOS-KVM#6] [Questions and Support]
Everytime i upgrade inside MacOS and reset through MacOS open core stays updated with my updated config.plist. But when I shut down, everything returns back when i run boot.sh....

**Comment by Coopydood** [comment]
> Everytime i upgrade inside MacOS and reset through MacOS open core stays updated with my updated config.plist. But when I shut down, everything returns back when i run boot.sh....

Try changing 
`-drive id=OpenCore,if=none,snapshot=on,format=qcow2,file="$REPO_PATH/boot/OpenCore.qcow2"`
to
`-drive id=OpenCore,if=none,format=qcow2,file="$REPO_PATH/boot/OpenCore.qcow2"`

and
``-drive if=pflash,format=raw,readonly=on,file="$REPO_PATH/$OVMF_DIR/OVMF_CODE.fd"``
to
``-drive if=pflash,format=raw,file="$REPO_PATH/$OVMF_DIR/OVMF_CODE.fd"``
